src/functions/uploadToRSIS/application/data-mapper/__tests__/data-mapper.spec.ts

```diff
@@ -26,6 +27,14 @@ import {
 } from '@dvsa/mes-test-schema/categories/AM1';
 import { getFullyPopulatedDrivingFaults } from './helpers/cat-a-mod2/inputs/fully-populated-inputs';
 import { getCatAMod2MinimalInput } from './helpers/cat-a-mod2/inputs/minimal-inputs';
+import { getCatADI2MinimalInput } from './helpers/cat-adi2/inputs/minimal-inputs';
+import {
+  getADI2FullyPopulatedDrivingFaults,
+  getADI2FullyPopulatedSeriousFaults,
+  getADIFullyPopulatedDangerousFaults,
+} from './helpers/cat-adi2/inputs/fully-populated-inputs';
+import { CatADI2UniqueTypes } from '@dvsa/mes-test-schema/categories/ADI2';
+import TestData = CatADI2UniqueTypes.TestData;
```

Comment:
Not sure if going mad but don't think this is the the typical way to import TestData

Reply:
TestData isn't exported directly, so can only reference it via CatADI2UniqueTypes.

Reply:
removed import of test data and am accessing where needed directly from CatADI2UniqueTypes